index.html

```diff
@@ -1 +1,97 @@
+                <select name="role" id="dropdown-role">
+                    <option value="challenges">Challenges</option>
+                    <option value="projects">Projects</option>
+                    <option value="community">Community</option>
+                    <option value="open-source">Open Source</option>
+                </select>
+        
+                <br>
+                <label for="improve" id="improve">What would you like to see improved? (Check all that apply)</label><br>
+                <input type="checkbox" id="front-end-projects" name="front-end-projects" value="front-end-projects">
+                <label for="front-end-projects">Front-end projects</label><br>
+                <br>
+                <input type="checkbox" id="back-end-project" name="back-end-project" value="back-end-project">
+                <label for="back-end-project">Back-end projects</label><br>
+                <br>
+                <input type="checkbox" id="data visualization" name="data visualizations" value="data visualization">
+                <label for="data visualization">Data visualization</label><br>
+                <br>
+                <input type="checkbox" id="challenges" name="challenges" value="challenges">
+                <label for="challenges">Challenges</label><br>
+                <br>
+                <input type="checkbox" id="open source community" name="open source community" value="open source community">
+                <label for="open source community">Open source community</label><br>
+                <br>
+                <input type="checkbox" id="gitter help rooms" name="gitter help rooms" value="gitter help rooms">
+                <label for="gitter help rooms">Gitter help rooms</label><br>
+                <br>
+                <input type="checkbox" id="videos" name="videos" value="videos">
+                <label for="videos">Videos</label><br>
+                <br>
+                <input type="checkbox" id="City meet ups" name="City meet ups" value="City meet ups">
+                <label for="City meet ups">City meet ups</label><br>
+                <br>
+                <input type="checkbox" id="wiki" name="wiki" value="wiki">
+                <label for="wiki">wiki</label><br>
+                <br>
+                <input type="checkbox" id="forum" name="forum" value="forum">
+                <label for="forum">forum</label><br>
+                <br>
+                <input type="checkbox" id="additional sources" name="additional sources" value="additional sources">
+                <label for="additional sources">additional sources</label><br>
+                <br>
+        
+                <label for="comment">Any comments or suggestions?</label><br>
+                <input type="text" id="comment" name="comment"  placeholder="Enter your comment here">
+                
+                <br>
```

Comment:
Good work on the form but there are a few features you are required  to apply, i.e Carryout form Validation to check for Email format, Maximum characters , Post Method and so on.